blog.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Adoptable Dogs Page </title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+</head>
+<body>
+    <script src="script/script.js"> </script>
+  <header>
+    <div class="header">
+      <img class= "dogface" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/header-image.jpg" alt= "brown dog looking through cracked blinds with sunlight beaming through">
```

Comment:
These image links don't work. They will need to be relative links to your project. If you look on your gihub pages you will see all of your images are broken

Reply:
This part confused me. I noticed that after I pushed everything. How do I fix the links?

Reply:
You should use relative paths in all of your work, it should work on your computer, as well as on a server (like github pages). Hopefully your editor didn't autofill the c://.... part.

```
      <img class= "dogface" src="images/header-image.jpg" alt= "brown dog looking through cracked blinds with sunlight beaming through">
```

If you update your links to this, it should work. Nice descriptive alt tag!